src/edit_session.js

```diff
@@ -186,7 +190,33 @@ class EditSession {
     getDocument() {
         return this.doc;
     }
-
+    /**
+     * Get "widgetManager" from editor.session
+     * 
+     * @returns {LineWidgets} object
+     */
+    get widgetManager() {
```

Comment:
nitpick: the spacing/indent here is not fully following what's expected here which makes it a bit hard to read

Reply:
ACK